modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -5,6 +5,8 @@
  * stanford_profile_styles.module
  */
 
+use Drupal\Core\Form\FormStateInterface;
+
 /**
  * Implements hook_page_attachments().
```

Comment:
The `admin.css` file should be attached through this hook and I don't see it here. Please add to this hook the logic to attach the `stanford_profile_styles/admin` library when the administrative theme is present. 

https://drupal.stackexchange.com/questions/229370/custom-module-for-adding-css-file-in-admin-section

Example code:
```
$is_admin = \Drupal::service('router.admin_context')->isAdminRoute();
    if ($is_admin) {
      $attachments['#attached']['library'][] = 'admin_css/admin-css';
      return;
    }
```

Reply:
Thanks, Shea!